wiki/installing-and-updating-drivers.md

```diff
@@ -86,6 +86,10 @@ See [Reinstalling GPU drivers from the OEM's website](reinstalling-gpu-drivers#m
 ## Troubleshooting 🔧
 ### If you can access Windows:
 #### Verifying Driver Signatures
+::: Why you should do that
+Maybe you installed a non-official driver by error and can cause system issues.
```

Comment:
Verifying driver signatures ensures you are installing secure, untampered and correct drivers issued by your hardware manufacturer.

Reply:
Fixed, thanks.